manifests/kots-helm.yaml

```diff
@@ -8,10 +8,18 @@ spec:
   chart:
     name: carto
     chartVersion: 1.110.1
+  builder:
```

Comment:
ohh crap, I was saying WTF is doing CARTO builder here? :see_no_evil:

Reply:
😂